data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -379,6 +379,7 @@ spec:
                                 control plane nodes, the valid value is pd-ssd.
                               enum:
                               - pd-ssd
+                              - pd-balanced
```

Comment:
Put this above pd-ssd. I think it's the ordering.

Reply:
Ok